perses/rjmc/topology_proposal.py

```diff
@@ -1177,6 +1180,9 @@ def build_system(self, new_topology):
         new_system : openmm.System
             A system object generated from the topology
         """
+        if self.verbose: print('Generating System...')
```

Comment:
Can we use logging? I realize it's not the best thing, but it's better than `print` because that way the user can control the verbosity level very easily.